### PR TITLE
#1544 - Fixed Visibility Settings of Repeater Fields not synced-in on…

### LINKED ISSUE #1544: Visibility Settings of Repeater Fields not synced-in on Account Privacy
**Describe the bug**
Visibility Settings of Repeater Fields not synced-in on Account Privacy

**To Reproduce**
1. Create a Profile Field set, Repeater Set = Enabled
2. Add a single field, Visibility = Public, Allow Members to override
3. Go to Profile > Edit Profile > Select the profile set create
4. Add information, then edit visibility to Only Me
5. Save Changes
6. Profile dropdown > Account > Privacy
7. Visibility does not sync up

**Expected behavior**
Synced-up visibility settings on **Profile Visibility on Edit Profile** and **Account >Privacy Settings**

**Screenshots**
https://www.loom.com/share/db188ed830ec42d69054f88a3f188231

**Support ticket links**
https://secure.helpscout.net/conversation/1280187196/98309

### PR DESCRIPTION
**Pull request by krunal-buddyboss**
… Account Privacy

### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #1544

### How to test the changes in this Pull Request:

    1. Create a Profile Field set, Repeater Set = Enabled
    2. Add a single field, Visibility = Public, Allow Members to override
    3. Go to Profile > Edit Profile > Select the profile set create
    4. Add information, then edit visibility to Only Me
    5. Save Changes
    6. Profile dropdown > Account > Privacy
    7. Visibility does not sync up

### Proof Screenshots or Video

https://tinyurl.com/y4bnmg4k

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-templates/bp-nouveau/buddypress/members/single/settings/profile.php [hunk 1]
```diff
@@ -23,6 +23,11 @@
 		<?php
 		while ( bp_profile_groups() ) :
 			bp_the_profile_group();
+			$group_id	= bp_get_the_profile_group_id();
+			// Check if Current Group is repeater if YES then get number of fields inside current group.
+			$is_group_repeater_str = bp_xprofile_get_meta( $group_id, 'group', 'is_repeater_enabled', true );
+			$is_group_repeater     = ( 'on' === $is_group_repeater_str ) ? true : false;
+			$group_url	= esc_url( trailingslashit( bp_displayed_user_domain() . bp_get_profile_slug() . '/edit/group/' . $group_id ) );
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/members/single/settings/profile.php:30]: @krunal-buddyboss Make a tabbing proper follow phpcs.

File: src/bp-templates/bp-nouveau/buddypress/members/single/settings/profile.php [hunk 2]
```diff
@@ -44,7 +49,13 @@
 
 							<tr <?php bp_field_css_class(); ?>>
 								<td class="field-name"><?php bp_the_profile_field_name(); ?></td>
-								<td class="field-visibility"><?php bp_profile_settings_visibility_select(); ?></td>
+								<?php if ( $is_group_repeater ) : ?>
+									<td class="field-visibility">
+										<a title="<?php esc_html_e( 'Manage Privacy', 'buddyboss' ); ?>" href="<?php echo $group_url; ?>" ><?php esc_html_e( 'Manage Privacy', 'buddyboss' ); ?></a>
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/members/single/settings/profile.php:54]: @krunal-buddyboss 
```suggestion
										<a title="<?php esc_html_e( 'Manage Privacy', 'buddyboss' ); ?>" href="<?php echo esc_url( $group_url ); ?>" ><?php esc_html_e( 'Manage Privacy', 'buddyboss' ); ?></a>
```

**Review by chetansatasiya — APPROVED** [review]
@krunal-buddyboss Do the requested changes

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
@krunal-buddyboss Do the requested changes

File: src/bp-templates/bp-nouveau/buddypress/members/single/settings/profile.php (hunk 1 above)
Comment (krunal-buddyboss) [reply src/bp-templates/bp-nouveau/buddypress/members/single/settings/profile.php]: @chetansatasiya  applied the changes that suggested. Thanks

**Review by mehulkaklotar — CHANGES_REQUESTED** [review]
@krunal-buddyboss it is still has phpcs errors. please redo them. and fix conflicts as well.

**Comment by krunal-buddyboss** [comment]
@mehulkaklotar  
I have fixed phpcs errors and also fixed conflicts as well. Thanks.

**Review by chetansatasiya — APPROVED** [review]

**Comment by mohit-buddyboss** [comment]
@krunal-buddyboss 

FIXED> Tested on Dev branch. Now for repeater field set logged in user getting Manage privacy option under Account>Privacy . So, now user will get option to manage privacy from one place.
Video : https://www.loom.com/share/3c7d37741c694263ad55b0b043abd4ac

Related issue is resolve now and PR is ready to merge.

cc @chetansatasiya

**Pull request merged**